utils.sh

```diff
@@ -102,18 +102,29 @@ function master_node_to_tf() {
     root_gb="$4"
     root_device="$5"
 
+    driver=$(master_node_val ${master_idx} "driver")
+    if [ $drive == "ipmi" ] ; then
```

Comment:
s/drive/driver - I think this is causing the CI failure

```
Mar 29 00:46:08 |  ++ jq -r '.nodes[0].driver' ocp/master_nodes.json
Mar 29 00:46:08 |  + driver=ipmi
Mar 29 00:46:08 |  utils.sh: line 106: drive: unbound variable
Mar 29 00:46:08 |  make: *** [ocp_run] Error 1
Last login: Thu Feb 28 21:57:05 UTC 2019 on pts/0
```

Reply:
thanks, that was a last minute typo as I tried to get this working on virt, will update once I make sure there is nothing else.